### DISCUSSION TITLE
No response for my Support Ticket for 25 Days.

### DISCUSSION THREAD

**Discussion by shubhankartrivedi** [community/community#52780] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have not got any response for my support ticket since a month.
I had a really critical issue which I had for a Year now but I ignored it.
But now as I wanna fix it, I opened a support ticket but sent like three messages every 10 days.

I have saw other waiting for years but it's very bad when someone have a very critical issue on his work account.

**Comment by ghost** [comment]
Did you get a response? Mine got suspended too, been more than a month and still no response.

**Reply by ghost** [reply]
ticket: #2719793
